tests/modules/lint.py

```diff
@@ -8,7 +8,7 @@ def test_modules_lint_trimgalore(self):
     module_lint.lint(print_results=False, module="trimgalore")
     assert len(module_lint.passed) > 0
     assert len(module_lint.warned) >= 0
-    assert len(module_lint.failed) == 0
+    assert len(module_lint.failed) == 0, f"Linting failed with {[x.__dict__ for x in module_lint.failed]}"
```

Comment:
Why was this added?

Reply:
The trimgalore test failed without error message in this PR. By adding this I could at least tell what the actual linting errors are. 

It's certainly not the most elegant solution to solve this but it helps. - LMK if I should remove it again!

Reply:
These failures should be printed somewhere no? In the results of `pytest`? Can you send me a screenshot? Wondering whether this is more of a systematic problem for the other linting code too.

Reply:
this is the full output of the CI: 
https://github.com/nf-core/tools/runs/3686704613

Unless I missed it, it only says that the lint failed, but not why. 

In any case, I should probably open an issue and remove the addition from this PR.

Reply:
Yup, ok. Good point. Let's leave it in for now.